### PR TITLE
Adds 2 mining vouchers to QM's locker

### PR DESCRIPTION
**Pull request by FluffySurvivor**
#920

### CODE REVIEW

**Comment by FluffySurvivor** [comment]
-.-

File: html/changelogs/FluffySurvivor-MiningVouchersQM.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+author: FluffySurvivor
++
++
++delete-after: True
++
++
++changes: 
++- rscadd: "Added 2 mining vouchers to the Quartermaster's locker."
```
Comment (X-TheDark) [inline html/changelogs/FluffySurvivor-MiningVouchersQM.yml]: This is because there's all these "+" in it. Remove them.

Comment (X-TheDark) [inline html/changelogs/FluffySurvivor-MiningVouchersQM.yml]:
```
author: FluffySurvivor

delete-after: True

changes: 
  - rscadd: "Added 2 mining vouchers to the Quartermaster's locker."
```

Comment (FluffySurvivor) [inline html/changelogs/FluffySurvivor-MiningVouchersQM.yml]: Wew, thank you !

File: html/changelogs/FluffySurvivor-MiningVouchersQM.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+author: FluffySurvivor
+
+
+delete-after: True
+
+
+changes: 
+ - rscadd: "Added 2 mining vouchers to the Quartermaster's locker."
```
Comment (X-TheDark) [inline html/changelogs/FluffySurvivor-MiningVouchersQM.yml]: Needs an extra space before "-". You need to indent with TWO spaces.

**Comment by FluffySurvivor** [comment]
God bless.

**Comment by ToGWtF** [comment]
why two though

**Comment by X-TheDark** [comment]
EDIT: Misunderstanding.

**Comment by ToGWtF** [comment]
No no,  I was asking why add two mining vouches lmao

**Comment by FluffySurvivor** [comment]
why one, considering how many miners you can have

**Comment by ghost** [comment]
well, it does allow a miner to grab two KA's, and go absolutely fuck-ham on anything they see

or grab adv scanner + KA from the start

**Comment by FluffySurvivor** [comment]
It's not like minedrones aren't ultra efficient anyway, a miner on steroids wouldn't be that big of a deal tbh. 

and a miner can't take the tickets by himself, he has to bug the shit out of the QM.

**Comment by EMT321** [comment]
This is a good thing for newer miners. I've seen a trend of people mining sand until they can get an advanced scanner, and only THEN going to mine.

**Comment by DavidEdwards** [comment]
I'm on the side with this. But we can try it.

As X-TheDark said:

> Needs an extra space before "-". You need to indent with TWO spaces.

**Comment by FluffySurvivor** [comment]
Kn0s, I already fixed that.

**Comment by DavidEdwards** [comment]
No you didn't.

**Comment by FluffySurvivor** [comment]
MY BAD

**Comment by DavidEdwards** [comment]
Bueno

**Pull request merged**